frontend/packages/process-editor/src/components/ConfigPanel/ConfigEndEvent/ConfigEndEvent.tsx

```diff
@@ -34,8 +34,11 @@ export const ConfigEndEvent = () => {
                   {t('process_editor.configuration_panel_custom_receipt_default_receipt_info')}
                 </Paragraph>
                 <Link
-                  href={altinnDocsUrl('app/development/configuration/process/customize/#receipt')}
+                  href={altinnDocsUrl(
+                    'nb/altinn-studio/reference/configuration/process/customize/#receipt',
```

Comment:
When using the Norwegian docs the link to the correct section is `#kvittering-receipt`